src/components/EmojiPicker/index.js

```diff
@@ -130,70 +108,38 @@ class EmojiPicker extends React.Component {
-                }
-                <Popover
-                    isVisible={this.state.isEmojiPickerVisible}
-                    onClose={this.hideEmojiPicker}
-                    onModalShow={this.focusEmojiSearchInput}
-                    onModalHide={this.props.onModalHide}
-                    hideModalContentWhileAnimating
-                    animationInTiming={1}
-                    animationOutTiming={1}
-                    anchorPosition={{
-                        bottom: this.props.windowHeight - this.state.emojiPopoverAnchorPosition.vertical,
-                        left: this.state.emojiPopoverAnchorPosition.horizontal - CONST.EMOJI_PICKER_SIZE,
-                    }}
-                >
-                    <EmojiPickerMenu
-                        onEmojiSelected={this.selectEmoji}
-                        ref={el => this.emojiSearchInput = el}
-                        preferredSkinTone={this.props.preferredSkinTone}
-                        updatePreferredSkinTone={this.setPreferredSkinTone}
-                        frequentlyUsedEmojis={this.props.frequentlyUsedEmojis}
-                    />
-                </Popover>
-                <Pressable
-                    style={({hovered, pressed}) => ([
-                        styles.chatItemEmojiButton,
-                        StyleUtils.getButtonBackgroundColorStyle(getButtonState(hovered, pressed)),
-                    ])}
-                    ref={el => this.emojiPopoverAnchor = el}
-                    onLayout={this.measureEmojiPopoverAnchorPosition}
-                    onPress={this.showEmojiPicker}
-                    disabled={this.props.isDisabled}
-                >
-                    {({hovered, pressed}) => (
-                        <Tooltip text={this.props.translate('reportActionCompose.emoji')}>
-                            <Icon
-                                src={Expensicons.Emoji}
-                                fill={StyleUtils.getIconFillColor(getButtonState(hovered, pressed))}
-                            />
-                        </Tooltip>
-                    )}
-                </Pressable>
-            </>
+            <PopoverWithMeasuredContent
+                isVisible={this.state.isEmojiPickerVisible}
+                onClose={this.hideEmojiPicker}
+                onModalShow={this.focusEmojiSearchInput}
+                onModalHide={this.onModalHide}
+                hideModalContentWhileAnimating
+                animationInTiming={1}
+                animationOutTiming={1}
+                anchorPosition={{
+                    vertical: this.state.emojiPopoverAnchorPosition.vertical,
+                    horizontal: this.state.emojiPopoverAnchorPosition.horizontal,
+                }}
+                popoverDimensions={{
+                    width: CONST.EMOJI_PICKER_SIZE,
+                    height: CONST.EMOJI_PICKER_SIZE,
```

Comment:
EMOJI_PICKER_SIZE indicates the width of it which can't be used for height. we have to give a static to the picker and create another variable for the height. Could you please ask on the slack what should be the height of the picker? I think 380 - 400 px looks good to me. 

we can define Const as: 
```js
EMOJI_PICKER: {
WIDTH: 320,
HEIGHT: 380
}

then apply the height to Picker and use to pass to the dimensions.

Reply:
It seems the height was 405 for the Popover. I've used 400 and it looks fine. Do we still need to check on Slack?

Reply:
Yes, please. I am not a design expert. @shawnborton Could you please help us determine the correct height for EmojiPicker?

Reply:
Hmm I'm honestly not too sure, but 400 seems fine to me?

Reply:
I think 400 is fine too.

Reply:
> take care of the case where screen height is less than 400 then it should just take screen height as before and respect all the margins that are applied

I am just testing everything right now.

Reply:
> then it should just take screen height as before

Was it working before? Afai can tell, the current picker still gets cut when the screen height is less than 400 on the web. For mobile devices, anyway, we have a bottom docked modal.